csv2docx/csv2docx.py

```diff
@@ -1,12 +1,44 @@
 import csv
 from mailmerge import MailMerge
-def convert(data, template, delimiter=";"):
+from pathlib import Path
+
+def create_path_if_not_exists(path: str):
+    """Creates a path to store output data if it does not exists.
+    Args:
+        path: the path from user in any format (relative, absolute, etc.)
+    Returns:
+        A path to store output data.
+    """
+    path = Path(path)
+    if not path.exists():
+        path.mkdir(parents=True, exist_ok=True)
+    return path
+
+def generate_name(filename, path):
+    """Checks whether file with same name exists or not
+    Args:
+        filename: the file we want to check
+        path: the directory we want to check in
+    Returns:
+        An unique full path with directory.
+    """
+    filename += ".docx"
+    checkpath = path / filename
+    if checkpath.exists():
+        # Count available files with same name
+        counter = len(list(path.glob(checkpath.stem + "*docx" ))) + 1
+        return f"{path}/{checkpath.stem}_{counter}.docx"
+    return checkpath
+
+def convert(data, template, delimiter=";", custom_name=None, path="output"):
     print ("Getting .docx template and .csv data files ...")
 
     with open(data, 'rt') as csvfile:
         csvdict = csv.DictReader(csvfile, delimiter=delimiter)
         csv_headers = csvdict.fieldnames
-
+        if (custom_name != None and custom_name not in csv_headers):
```

Comment:
This is E711 as identified by flake8. You can write `custom_name` instead of `custom_name != None

Reply:
now that we made custom_name required, we don't need that condition.

Reply:
Great point -- let's remove the IF statement 👍